app/policies/organisation_policy.rb

```diff
@@ -4,6 +4,20 @@ def index?
   end
 
   def show?
-    true
+    user.administrator? || user.organisations.include?(@record)
+  end
+
+  def update?
+    user.administrator? || user.organisations.include?(@record)
+  end
+
+  class Scope < Scope
```

Comment:
This will get ignored because of the removal of `policy_scope` in `Staff::OrganisationController`